CWE/inventions/New inventions.txt

```diff
@@ -300,4 +300,465 @@ full_body_replacement = {
             health = 0.05
         }
 	}
+}
+
+land_ethic = {
+	limit = { intensive_farming = 1 }
+	chance = {
+		base = 1
+		modifier = {
+			factor = 2
+			year = 1980
+		}
+	}
+
+	effect = {
+		shared_prestige = 5
+		farm_rgo_eff = 0.01 #More efficient/sustainable use of land
+	}
+}
+
+smart_grid = {
+	limit = { sustainable_energy = 1 }
+	chance = {
+		base = 1
+		modifier = {
+			factor = 2
+			handheld_devices = 1
+		}
+		modifier = {
+			factor = 2
+			modern_backbone = 1
+		}
+	}
+	effect = {
+		factory_goods_output = {
+			heating_electricity = 0.05 
+		}
+	}
+}
+
+techno_progressivism = {
+	limit = { seventies_subcultures = 1 }
+	chance = {
+		base = 1
+		modifier = {
+			factor = -5
+			NOT = { strd_internet = 1 }
+		}
+		modifier = {
+			factor = 1
+			NOT = { OR = { government = proletarian_dictatorship government = proletarian_dictatorship1 government = nationalist_dictatorship government = populist_dictatorship government = absolute_monarchy government = presidential_dictatorship } }
+		}
+		modifier = {
+			factor = 2
+			OR = { ruling_party_ideology = progressive ruling_party_ideology = socialist }
+		}
+	}
+	effect = {
+		administrative_efficiency = 0.010
+	}
+}
+
+cyborg_anthropology = {
+	limit = { informatization_education = 1 }
+	chance = {
+		base = 1
+		modifier = {
+			factor = 1
+			virtual_science = 1
+		}
+		modifier = {
+			factor = -1
+			ruling_party_ideology = traditionalist
+		}
+	}
+	effect = {
+		education_efficiency = 0.01
+	}
+}
+
+
+dark_enlightenment = {
+	limit = { age_of_protests = 1 }
+	chance = {
+		base = 1
+		modifier = {
+			factor = 1
+			OR = { government = democracy government = democracy1 }
+		}
+		modifier = {
+			factor = 1
+			OR = { ruling_party_ideology = traditionalist ruling_party_ideology = populist }
+		}
+		modifier = {
+			factor = -1
+			NOT = { OR = { continent = europe continent = north_america } }
```

Comment:
capital_scope = { NOT = { OR = { continent = europe continent = north_america } } }